### ISSUE TITLE
Can't find tanh function in eqn. 8  

### ISSUE THREAD

**Issue by Bintodo** [Seanny123/da-rnn#3]
![image](https://user-images.githubusercontent.com/11547963/49231735-852fad00-f42d-11e8-8744-40811e9df8d4.png)
like above, i can't find tanh function.

**Comment by josephstalin117** [comment]
May be use another attention layer

**Comment by ankur6ue** [comment]
yep.. The Wc and U matrices in eq8 in the original paper are missing as well.
